src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+mechanical objects that are part of mechanical processes. From this paragraph
+alone, it is not clear whether Hegel thinks that laws or morals are examples of
+`communication` or that he thinks that _they can be_ used or expressed in the
+form of `communication`. I suspect that he thinks that the proliferation of
+laws, customs, etc, are logically structured according to `communication`. See:
+Nathan Ross. Let us have a look at the first sentence of this paragraph to get a
+better understanding of what Hegel is going for:
+
+> Now in the spiritual sphere there is an infinitely manifold content that is
+> communicable; for being taken up into intelligence it receives this _form_ of
+> universality in which it becomes communicable. But the universal that is such
+> not merely through the form but in and for itself, is the _objective_ as such,
+> both in the spiritual and in the material sphere; as against which the
+> individuality of outer objects as well as of persons is an unessential element
+> that can offer it no resistance (Hegel 1991, 716).
+
+### How Communication is different to Intelligence
+
+The paragraph begins with an example that does not fall within the purview of
+`communication` because it is concerned with “intelligence” (Hegel 1991, 716).
+Intelligence, then, does not fall within the logical remit of `Mechanism`. If we
+just focus on the text, however, it looks as if intelligence has the effect of
+giving the form of universality to a content and thereby making it communicable.
+There are 2 logical steps to clarify here: (1) intelligence takes on a manifold
+content and gives it the form of universality – a capacity that intelligence
+has, and (2) intelligence is able to communicate the content _because_ it has
+the form of universality, _i.e._ a communicated content, whether it be through
+the `formal mechanical process` or through intelligence, must be universalised
+in order to be communicated.[^4]: It would be a fascinating thread of further
+research to explain exactly why intelligence has the capacity to give the form
+of universality to manifold content. I suspect that a good place to start for
+such an enquiry would be the ‘Philosophy of Spirit’.
+
+In contrast to intelligence, processes within the objective sphere, _i.e._ the
+`formal mechanical process` do not give to content the form of universality but
+are in-and-for-themselves universal. Why is this the case? The answer must have
+to do with the very beginning of `Mechanism` where the `mechanical object`.
+Hegel writes that “It is therefore in and for itself a universal -universality
+not in the sense of a community of properties, but a universality that pervades
+the particularity and in it is immediate individuality” (Hegel 1991, 711). The
+reason why the objective, and by extension the `formal mechanical process` is
+in-and-for-itself the universal is because the difference between the
+determinations of `Concept` have been effaced.[^5]: See:
```

Comment:
Should we capitalize the concept?

Reply:
I think we should capitalise it because it refers to the section heading. Moreover, I think backticks help to crystallise it as a word that refers to the text and not an extra-textual word. So it's technical meaning, rather than common use meaning.

Reply:
But if you refer to a section heading, you refer to a piece of text, _not_ the concept 😛